### PR TITLE
retry on 'Lost connection' also

### PR DESCRIPTION
**Pull request by pulkitjalan**
Auto retry if 'Lost connection to MySQL server ' as well

### CODE REVIEW

File: src/Illuminate/Database/Connection.php [hunk 1]
```diff
@@ -658,7 +658,11 @@ protected function causedByLostConnection(QueryException $e)
 	{
 		$message = $e->getPrevious()->getMessage();
 
-		return str_contains($message, ['server has gone away', 'no connection to the server']);
+		return str_contains($message, [
+			'server has gone away',
+			'no connection to the server',
+			'Lost connection'
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: Missing trailing comma.

Comment (pulkitjalan) [inline src/Illuminate/Database/Connection.php]: oops, should I do another PR for it?

Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: you can do, it's probably not worth the effort though

Comment (pulkitjalan) [inline src/Illuminate/Database/Connection.php]: Added #7684
